packages/smooth_app/lib/themes/color_schemes.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+const Color seed = Color(0xFF99460D);
+
+const ColorScheme lightColorScheme = ColorScheme(
+  brightness: Brightness.light,
+  primary: Color(0xFF341100),
+  onPrimary: Color(0xFFFFFFFF),
```

Comment:
From what I saw in the Figma screenshots (cf. app bar)
* `primary` is white
* `onPrimary` is black